### ISSUE TITLE
[8.0] Wrong `onchange_project` signature in project.task prevents new api use

### ISSUE THREAD

**Issue by boolit-eivydas** [odoo/odoo#12856]
Impacted versions:
8.0

Steps to reproduce:
Extend the method with `@api.multi`, call the parent method with `super()`.
Go to a new task form view, change the project. The parent method is not called.

Current behavior:
Extending the method with `@api.multi`, the `super()` method does not find the parent implementation when called from a new task view. The already existing tasks do trigger the behavior correctly.

After changing the signature to have `ids` instead of `id`, everything works as intended.
`def onchange_project(self, cr, uid, ids, project_id, context=None):`

Expected behavior:
On both new and existing tasks, when the `project_id` is changed, all extending methods have access to parent method.

**Comment by mart-e** [comment]
Do you mean something like:

```python
from openerp import models, api

class project_task2(models.Model):
    _inherit = 'project.task'

    @api.multi
    def onchange_project(self, project_id):
        print "doing stuff"
        return super(project_task2, self).onchange_project(project_id)
```

It seems to work fine on my side

**Comment by boolit-eivydas** [comment]
Could you also print the return value of the `super()` here?

**Comment by mart-e** [comment]
ok I see the issue, the return format was wrong indeed
Fixed that at 8b83119fad7ccae9f091f12b6ac89c2c31e4bac3
Thanks for the report
